x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringService.java

```diff
@@ -104,6 +126,14 @@ public boolean isMonitoringActive() {
         return isStarted() && enabled;
     }
 
+    public boolean isElasticsearchCollectionEnabled() {
+        return this.elasticsearchCollectionEnabled;
+    }
+
+    public boolean shouldScheduleExecution() {
```

Comment:
Should this and some of the other related methods in here be private or at least protected?

Reply:
I'd be okay with either way. I even though about package protected after I initially wrote it (so it can be tested but not reused). But you're right it doesn't need to be `public`.